cmd/cmd_pkg.go

```diff
@@ -228,6 +228,13 @@ func loadCommandHandler(cmd *cobra.Command, args []string) {
 		exitWithError(err.Error())
 	}
 
+	packageFolder := filepath.Base(pkgFlags.PackagePath())
+	executableName := c.Program
+	if strings.Contains(executableName, packageFolder) {
+		executableName = filepath.Base(executableName)
+	}
+	lepton.ValidateELF(filepath.Join(pkgFlags.PackagePath(), executableName))
```

Comment:
This breaks loading of packages (e.g. java_1.8.0_191 and java_11.0.8) where the executable file is inside the sysroot folder.
For example, trying to load java_1.8.0_191 fails with `2021/05/16 12:26:20 open /home/francesco/.ops/packages/java_1.8.0_191/usr/lib/jvm/java-8-openjdk-amd64/jre/bin/java: no such file or directory` (the correct path of the program executable is /.ops/packages/java_1.8.0_191/sysroot/usr/lib/jvm/java-8-openjdk-amd64/jre/bin/java)

Reply:
https://github.com/nanovms/ops/issues/1076